tests/test_feedexport.py

```diff
@@ -969,3 +984,341 @@ def test_pathlib_uri(self):
         }
         data = yield self.exported_no_data(settings)
         self.assertEqual(data['csv'], b'')
+
+
+class PartialDeliveriesTest(FeedExportTestBase):
```

Comment:
Here and in a few other parts of the code it says “partial”; ”batch” is more consistent with the rest of the code, and more unambiguous (“partial” may be interpreted as not delivering all data).

```suggestion
class BatchDeliveriesTest(FeedExportTestBase):
```